src/main/java/de/unistuttgart/overworldbackend/service/KeybindingService.java

```diff
@@ -0,0 +1,112 @@
+package de.unistuttgart.overworldbackend.service;
+
+import de.unistuttgart.overworldbackend.data.Keybinding;
+import de.unistuttgart.overworldbackend.data.KeybindingDTO;
+import de.unistuttgart.overworldbackend.data.Player;
+import de.unistuttgart.overworldbackend.data.enums.Binding;
+import de.unistuttgart.overworldbackend.repositories.KeybindingRepository;
+import de.unistuttgart.overworldbackend.repositories.PlayerRepository;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.server.ResponseStatusException;
+
+@Service
+@Transactional
+public class KeybindingService {
+
+    @Autowired
+    private PlayerRepository playerRepository;
+
+    @Autowired
+    private KeybindingRepository keybindingRepository;
+
+    /**
+     * Checks for all players the current keybindings, adds new created keynindings to the player and removes none existing keybindings.
+     */
+    @EventListener(ApplicationReadyEvent.class)
+    public void updatePlayerKeybindings() {
+        Binding[] bindings = Binding.values();
+        for (final Player player : playerRepository.findAll()) {
+            for (Binding binding : bindings){
+                if (
+                        player
+                            .getKeybindings()
+                            .stream()
+                            .noneMatch(keybinding ->
+                                    keybinding
+                                        .getBinding()
+                                        .equals(binding))
+                ) {
+                    player.getKeybindings().add(new Keybinding(player, binding, ""));
```

Comment:
How does the player make an action when the binding is empty? Is the frontend checking this part, when binding is empty, that the frontend tells the backend the default key?